spec/acceptance/firewall_attributes_happy_path_spec.rb

```diff
@@ -350,6 +349,11 @@ class { '::firewall': }
             chain          => 'OUTPUT',
             table          => 'mangle',
           }
+          firewall { '1002 - set ipvs':
```

Comment:
How this test tests ipvs?

Reply:
@mateusz-gozdek-sociomantic Nicely spotted! Tt seems I forgot to actually add the full test. I've amended this in my PR #833 and fixed an idempotency issue I uncovered.

Reply:
You're welcome, cool that it's fixed now!